gammapy/scripts/image_pipe.py

```diff
@@ -101,6 +102,42 @@ def bkg_map(self, bkg_norm=True):
 
         self.maps["bkg"] = bkg_map
 
+    def make_1d_expected_counts(self, spectral_index=2.3, for_integral_flux=False):
+        """Compute the 1D exposure table for one observation for an offset table
+
+        Parameters
+        ----------
+        spectral_index : float
+            Assumed power-law spectral index
+        for_integral_flux : bool
+            True if you want that the total excess / exposure gives the integrated flux
+
+        Returns
+        -------
+        table: `astropy.table.Table`
```

Comment:
Missing whitespace before the colon, should be:
```
table : `astropy.table.Table`
```
Please always check the HTML version of new docstrings you've written locally using:
```
python setup.py build_sphinx
open docs/_build/html/index.html
```